atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -251,7 +291,11 @@ def _are_features_enabled(self):
         return pin_digest, replace_repo, replace_registry
 
     def _set_worker_arg(self, replacement_pullspecs):
-        arg = {str(old): str(new) for old, new in replacement_pullspecs.items()}
+        arg = {
+            str(repl['original']): str(repl['new'])
```

Comment:
no need to convert it to str anymore since we will have strings from _get_replacement_pullspecs already